<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastroPagamento</class>
 <widget class="QDialog" name="CadastroPagamento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1424</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Pagamento</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabCadastrarPag">
      <attribute name="title">
       <string>Cadastrar pagamentos</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_18">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QFrame" name="frame_3">
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="3">
             <widget class="TableView" name="tableTaxas">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButtonRemoverPagamento">
              <property name="text">
               <string>Remover pagamentos</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButtonLimparSelecao">
              <property name="text">
               <string>Limpar seleção</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="pushButtonAtualizarTaxas">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Atualizar taxas</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" rowspan="2">
             <widget class="QLabel" name="label_8">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>&gt;&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="TableView" name="tablePagamentos">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="2">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Conta Destino</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="spinBoxParcelas">
                <property name="minimum">
                 <number>1</number>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Parcelas</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Pagamento</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="lineEditPagamento"/>
              </item>
              <item row="1" column="2">
               <widget class="ItemBox" name="itemBoxContaDestino"/>
              </item>
             </layout>
            </item>
            <item row="0" column="1" rowspan="2">
             <widget class="QPushButton" name="pushButtonAdicionarPagamento">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Adicionar pagamento</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" rowspan="2">
             <widget class="QPushButton" name="pushButtonAtualizarPagamento">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Atualizar pagamento</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="1">
               <widget class="QCheckBox" name="checkBoxDMaisUm">
                <property name="text">
                 <string>D+1</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QCheckBox" name="checkBoxApenasRepresentacao">
                <property name="text">
                 <string>Apenas Representação</string>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="3">
               <widget class="QCheckBox" name="checkBoxPula1Mes">
                <property name="text">
                 <string>Pula 1º mês</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QCheckBox" name="checkBoxDiaUtil">
                <property name="text">
                 <string>Ajuste Dia Útil</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QCheckBox" name="checkBoxCentavoSobressalente">
                <property name="text">
                 <string>Centavos na 1ª parcela</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAssociarPag">
      <attribute name="title">
       <string>Associar loja-pagamento</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Loja:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ItemBox" name="itemBoxLoja"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Pagamentos não associados</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_14">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Pagamentos associados</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="TableView" name="tableAssocia1">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButtonAdicionaAssociacao">
           <property name="text">
            <string>Associar &gt;&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButtonRemoveAssociacao">
           <property name="text">
            <string>Desassociar &lt;&lt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="2">
        <widget class="TableView" name="tableAssocia2">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ContiguousSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabComissao">
      <attribute name="title">
       <string>Comissão</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QFrame" name="frame_4">
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Comissão loja:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBoxLoja"/>
            </item>
            <item row="0" column="2">
             <widget class="ItemBox" name="itemBoxContaLoja">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Comissão profissional:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBoxProfissional"/>
            </item>
            <item row="1" column="2">
             <widget class="ItemBox" name="itemBoxContaProfissional">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Comissão vendedor:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="comboBoxVendedor"/>
            </item>
            <item row="2" column="2">
             <widget class="ItemBox" name="itemBoxContaVendedor">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>src/tableview.h</header>
  </customwidget>
  <customwidget>
   <class>ItemBox</class>
   <extends>QLineEdit</extends>
   <header>src/itembox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget_2</tabstop>
  <tabstop>lineEditPagamento</tabstop>
  <tabstop>spinBoxParcelas</tabstop>
  <tabstop>itemBoxContaDestino</tabstop>
  <tabstop>checkBoxDiaUtil</tabstop>
  <tabstop>checkBoxDMaisUm</tabstop>
  <tabstop>checkBoxCentavoSobressalente</tabstop>
  <tabstop>checkBoxPula1Mes</tabstop>
  <tabstop>checkBoxApenasRepresentacao</tabstop>
  <tabstop>pushButtonAdicionarPagamento</tabstop>
  <tabstop>pushButtonAtualizarPagamento</tabstop>
  <tabstop>tablePagamentos</tabstop>
  <tabstop>tableTaxas</tabstop>
  <tabstop>pushButtonRemoverPagamento</tabstop>
  <tabstop>pushButtonLimparSelecao</tabstop>
  <tabstop>pushButtonAtualizarTaxas</tabstop>
  <tabstop>itemBoxLoja</tabstop>
  <tabstop>tableAssocia1</tabstop>
  <tabstop>tableAssocia2</tabstop>
  <tabstop>pushButtonAdicionaAssociacao</tabstop>
  <tabstop>pushButtonRemoveAssociacao</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
